validation-test/Sema/type_checker_perf/fast/more_specialized_generic_func.swift.gyb

```diff
@@ -1,4 +1,4 @@
-// RUN: %scale-test --begin 1 --end 10 --step 1 --select incrementScopeCounter %s --expected-exit-code 0
+// RUN: %scale-test --begin 8 --end 40 --step 2 --select incrementScopeCounter %s --expected-exit-code 0
```

Comment:
Unfortunately this changes made this test fail to fit the curve once again, so I had to play with numbers a bit to make it work again...